src/main/java/com/Saurabh/GitHubAiAPI/StringOperations.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) {
 
         // Validate input
         if (input == null || input.isEmpty()) {
-            System.out.println("Invalid input. Please enter a non-empty string.");
+            System.out.println("Invald input. Please enter non-empty string.");
             return;
         }
 
```

Comment:
There is a spelling mistake in the error message. It should be 'Invalid input. Please enter a non-empty string.' instead of 'Invald input. Please enter non-empty string.'.